### PR TITLE
Add direct storage client

### PR DESCRIPTION
**Pull request by khorshuheng**
Signed-off-by: Khor Shu Heng <khor.heng@go-jek.com>

**What this PR does / why we need it**:
This PR provides direct storage client, which retrieve features directly from the underlying online storage of Feast rather than going through Feast Serving

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

- [ ] Added unit test, integration, and/or e2e tests
- [ ] Tested locally
- [ ] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

File: api/cmd/transformer/main.go [hunk 1]
```diff
@@ -39,8 +40,9 @@ type AppConfig struct {
 	Server server.Options
 	Feast  feast.Options
 
-	StandardTransformerConfigJSON string `envconfig:"STANDARD_TRANSFORMER_CONFIG" required:"true"`
-	LogLevel                      string `envconfig:"LOG_LEVEL"`
+	StandardTransformerConfigJSON string	`envconfig:"STANDARD_TRANSFORMER_CONFIG" required:"true"`
+	FeatureTableSpecJsons  		  []string  `envconfig:"FEAST_FEATURE_TABLE_SPECS_JSONS" default:"false"`
```
Comment (pradithya) [inline api/cmd/transformer/main.go:44]: Why `default:"false` ? shouldn't it be `required:"true"`?

Comment (pradithya) [reply api/cmd/transformer/main.go:44]: Can you fix formatting too, it doesn't seems to align correctly

File: api/pkg/transformer/feast/storage.go [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+	"github.com/go-redis/redis/v8"
+	"github.com/golang/protobuf/proto"
+	"github.com/spaolacci/murmur3"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"sort"
+	"strings"
+	"time"
+)
+
+func NewDirectStorageClient(storage *OnlineStorage, featureTables []*core.FeatureTableSpec) (StorageClient, error) {
```
Comment (tiopramayudi) [inline api/pkg/transformer/feast/storage.go:22]: forget one thing, how about credentials for redis like password? do we use one?

**Comment by ariefrahmansyah** [comment]
For formatting, please use `https://github.com/mvdan/gofumpt`

**Review by pradithya — APPROVED** [review]
LGTM! thanks @khorshuheng

**Pull request merged**